### ISSUE TITLE
Widgets in readonly state are incorrectly read as "editable" by NVDA

### ISSUE THREAD

**Issue by IvanDanchev** [telerik/kendo-ui-core#5444]
### Bug report
The **aria-readonly** attribute was removed from input elements with role "listbox" or "combobox": [https://github.com/telerik/kendo-ui-core/issues/1740](https://github.com/telerik/kendo-ui-core/issues/1740)
Currently, the WAI-ARIA specification lists these roles as compatible with the **aria-readonly** attribute: [https://www.w3.org/TR/wai-aria-1.1/#aria-readonly](https://www.w3.org/TR/wai-aria-1.1/#aria-readonly)
As a result of the attribute removal, the DateTimePicker and the ComboBox (and potentially other widgets that render an input element with one of these two roles) in readonly state are read as "editable" by NVDA.

### Reproduction of the problem
[Dojo](https://dojo.telerik.com/OrEKEfiF/3) example.
1. Run NVDA and focus the DateTimePickers and the ComboBox. 

### Current behavior
NVDA reads them as "editable", which is incorrect.

### Expected/desired behavior
The inputs should not be read as "editable"

### Environment

* **Kendo UI version:** 2019.3.1023
* **jQuery version:** x.y
* **Browser:** [all ]

**Comment by mparvanov** [comment]
http://kendobuild.telerik.com/dojo-staging/adiFAVIq 
Dropdownree is missing aria-readonly="true"
